chart/values.yaml

```diff
@@ -2505,6 +2505,9 @@ dags:
     # and specify the name of the secret below
     # sshKeySecret: airflow-ssh-secret
     #
+    # Or set sshKeySecret with your key
+    # sshKey: <base64-converted-ssh-private-key>
```

Comment:
And this. Probably worth putting an example with an empty key, since this is a multi-line deal.

Reply:
let me know if you ment like what I changed